src/main/java/frc/robot/Robot.java

```diff
@@ -39,7 +39,7 @@ public enum RobotId {
    * <p><b>Note: kRobotId is not a constant.</b></p>
    */
   // assume a default robot identity
-  public static RobotId kRobotId = RobotId.Default;
+  public static RobotId kRobotId = RobotId.SwerveTest;
```

Comment:
Are you planning to run this on test swerve?

Reply:
lol sowwy